samza-shell/src/main/bash/run-class.sh

```diff
@@ -69,10 +69,7 @@ if [ -d "$JOB_LIB_DIR" ] && [ "$JOB_LIB_DIR" != "$BASE_LIB_DIR" ]; then
   echo generated combined CLASSPATH=$CLASSPATH
 else
   #default behavior
-  for file in $BASE_LIB_DIR/*.[jw]ar;
-  do
-    CLASSPATH=$CLASSPATH:$file
-  done
+  CLASSPATH=$CLASSPATH:"$BASE_LIB_DIR/*";
```

Comment:
Looks good.
One question. Now we are going to pickup everything including *.xml. Is it a problem?

Reply:
From Java Docs: https://docs.oracle.com/javase/7/docs/technotes/tools/windows/classpath.html

Class path entries can contain the basename wildcard character *, which is considered equivalent to specifying a list of all the files in the directory with the extension .jar or .JAR. For example, the class path entry foo/* specifies all JAR files in the directory named foo. A classpath entry consisting simply of * expands to a list of all the jar files in the current directory.

This is based on assumption that we don't need any war files in classpath

Reply:
Just curious.

1. The control script previously was adding both the `jar` and `war` files to the `classpath`.  After this change, we're just adding only jar files into the `classpath`.  There're existing samza jobs in linkedin which use war files in classpath.  Do we know if this change will break the existing jobs?

2. In addition to wild carding jars, can we iterate and add *only* the wars to the classpath just to maintain backward compatibility? 

3.  Link cited in above comment refers to classpath expansion semantics for windows. Since we use linux extensively, do we know if the same behavior applies to linux as well?

Reply:
@shanthoosh thanks for pointing out, I updated with the link for linux and solaris